### PR TITLE
feat(assistant): add need help buttons in settings pages

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-1101

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Programs must be written for people to read, and only incidentally for machines to execute

File: libs/domains/service-helm/feature/src/lib/networking-setting/networking-setting.tsx [hunk 1]
```diff
@@ -58,41 +60,53 @@ export function NetworkingSetting({ ports, onUpdatePorts, children }: Networking
...
+      ) : (
+        <div className="flex flex-row w-full justify-between">
+          <div>
+            <Heading className="mb-2">Networking</Heading>
+            <p className="text-sm text-neutral-350 mb-2">
+              You can expose publicly over HTTP/gRPC the Kubernetes services deployed.
+            </p>
+            <Popover.Root>
+              <Popover.Trigger>
+                <span className="text-sm cursor-pointer text-brand-500 hover:text-brand-600 transition font-medium mb-5">
```
Comment (RemiBonnet) [inline libs/domains/service-helm/feature/src/lib/networking-setting/networking-setting.tsx:82]: You didn't want to add it on the button component, like we have talk on slack ?

Comment (ctjhoa) [reply libs/domains/service-helm/feature/src/lib/networking-setting/networking-setting.tsx:82]: This one is the legacy / existing design for the creation flow.
As my story is only about settings pages, I needed to differentiate settings from creation flow.
I did use the `Button` component for the `NeedHelp` component (the new one)

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**